dist/s_clang-scan

```diff
@@ -17,9 +17,9 @@ test -z "$scan" || compiler="${SCANCOMPILER:-clang}"
 
 if test -z "$scan" &&
     test -x /usr/bin/clang &&
-    test -x /usr/lib/llvm-3.9/bin/scan-build; then
+    test -x /usr/bin/scan-build; then
```

Comment:
For my understanding, could this change/assumption work well in non-ubuntu2004 environments e.g. a Ubuntu 18.04 local dev host, or a macOS host?

Reply:
No, I do not think that this change works well for non-ubuntu environments. I think it would be a good idea to add a conditional statement here. We can keep the location to /usr/buid/scan-build if the ubuntu release is 20.04. Does that sound good?

Reply:
Yeah, it's important the change won't break people's current workflow (e.g. running `dist/s_all`) in their local environments.

Reply:
I looked in the s_all script and it doesn't run the s_clang-scan script. This script is only run by the clang analyzer test.

Reply:
Thanks for checking. Then I have no concern with the current approach. Please test the `clang-analyzer` task on MSAN and UBSAN builders to make sure they work as expected.

Reply:
Yes, the `clang-analyzer ` task runs successfully on MSAN AND UBSAN.